webnn/resources/test_data/constant.json

```diff
@@ -0,0 +1,708 @@
+{
+  "tests": [
+    {
+      "name": "constant float32 0D tensor of default float32 type",
+      "inputs": {
+        "start": {
+          "data": 0.22992068529129028,
+          "type": "float32"
+        },
+        "step": {
+          "data": 0.7537541389465332,
+          "type": "float32"
+        }
+      },
+      "outputShape": [],
+      "expected": {
+        "name": "output",
+        "shape": [],
+        "data": [
+          0.22992068529129028
```

Comment:
Appears you already coerced the value to float32, because the input and output match exactly 👍.

```
      "inputs": {
        "start": {
          "data": 0.22992068529129028,
...
        },
      },
...
      "expected": {
...
          0.22992068529129028,
```